Zhuang/README.md

```diff
@@ -0,0 +1,33 @@
+# Yan Zhuang student-developer-setup
+
+1) Created fork on GitHub
+
+2) Created README.md
+
+3) Pushed to GitHub, created pull request
+
+4) Using Pycharm (Working on Windows System)
+
+5) Created `hello.py` as a "hello world" program; can be built via the `build.sh` script.
+6-7) Created requirement.txt and add pre-commit as required package
+8) Created pre-commit yaml configuration file using its sample official config file
+
+9) Installed docker
+
+10) Read the guide for how to use docker
+
+11) Created a Dockerfile Zhuang folder that builds and runs the "hello world" program
+
+    This Dockerfile can be used to compile and run the "Hello" program. First, make sure Docker is installed, then `cd` into the `battle` directory and run:
+
+    `docker build -t my_image_name .`
+
+    Then, use
+
+    `docker container run --rm my_image_name`
```

Comment:
```suggestion
    `docker run --rm my_image_name`
```

While it isn't wrong to use the [`container` CLI API](https://docs.docker.com/engine/reference/commandline/container/) to run containers, it is far more common to use the [`docker run` CLI API](https://docs.docker.com/engine/reference/commandline/run/) and so just for conforming to standards it is better to use it here. The `docker container` CLI is more common for controlling existing containers.

Reply:
fix it